### PR TITLE
logutil: InitLogger as a more abstract method, deprecated InitZapLogger

### PR DESCRIPTION
**Pull request by tisonkun**
Signed-off-by: tison <wander4096@gmail.com>

### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/25380

### What is changed and how it works?

What's Changed:

* Replace `InitZapLogger` with `InitLogger` and deprecated `InitZapLogger`
* Replace `pumpcli.InitLogger` with `logutil.InitLogger` where is an unnecessary middleman.
* Will remove `InitZapLogger` when https://github.com/pingcap/tidb-tools/pull/444 merged after this one merged.

### Release note

- No release note

### CODE REVIEW

**Comment by tisonkun** [comment]
cc @xhebox @lysu

File: util/logutil/log.go [hunk 1]
```diff
@@ -94,16 +94,11 @@ const (
 	OldSlowLogTimeFormat = "2006-01-02-15:04:05.999999999 -0700"
 )
 
-// SlowQueryLogger is used to log slow query, InitZapLogger will modify it according to config file.
+// SlowQueryLogger is used to log slow query, InitLogger will modify it according to config file.
 var SlowQueryLogger = log.L()
 
-// InitLogger delegates to InitZapLogger. Keeping it here for historical reason.
+// InitLogger initializes a zap logger with cfg.
```
Comment (tisonkun) [inline util/logutil/log.go:100]:
```suggestion
// InitLogger initializes a logger with cfg.
```

**Comment by tisonkun** [comment]
given that pingcap/tidb-tools#444  is merged, I'm gonna remove `InitZapLogger`.

**Review by xhebox — APPROVED** [review]
Looks like you haven't deprecate ZapLogger even after the merge of tidb-tools PR, but whatever.

**Review by lysu — APPROVED** [review]

**Pull request merged**